SPEC.md

```diff
@@ -455,36 +696,74 @@ lines](https://www.unicode.org/versions/Unicode13.0.0/ch05.pdf):
 
 Note that for the purpose of new lines, CRLF is considered _a single newline_.
 
+### Disallowed Literal Code Points
```

Comment:
Should BOM/ZWNBSP be included in this list? The changelog states that BOM is now prohibited except as the first character, but that is not reflected anywhere in the spec (which should imply that ZWNBSP is allowed in an identifier).

Reply:
Yeah, let's do that.